Sources/eppo/EppoClient.swift

```diff
@@ -1,32 +1,30 @@
 import Foundation;
 
-public let version = "1.0.1"
-
-public struct FlagConfigJSON : Decodable {
-    var flags: [String : FlagConfig];
+// todo: make this a build argument (FF-1944)
+public let sdkVersion = "1.0.1"
+public let sdkName = "ios"
```

Comment:
these are the parameters reported to the server - we have forgotten to update this constant in the last few releases. I will look for improved ways to populate the `sdkVersion` based on build arguments.